src/Stratis.Bitcoin.Features.Miner/MiningFeature.cs

```diff
@@ -158,7 +162,7 @@ public override void Initialize()
         /// <inheritdoc />
         public override void Dispose()
         {
-            this.powLoop?.Dispose();
```

Comment:
@noescape00 this is exactly what was happening anyway though?

Not that it's 'correct', just that the functionality hasn't changed.